src/styled.d.ts

```diff
@@ -0,0 +1,9 @@
+import 'styled-components';
+import { Colors, Size } from './globalStyle';
+
+declare module 'styled-components' {
+  export interface DefaultTheme {
+    colors: Colors;
+    size: Size;
+  }
```

Comment:
아 이거 찾았어요 지금 이게 너무 별로였는데 (왜냐하면 property 추가될때마다 여기도 추가해야하니까)
아래처럼 하면 해결됩니다
```suggestion
  export interface DefaultTheme extends typeof theme {}
```

typeof theme 을 theme.ts 에서 export 시켜주면 아래처럼 깔끔하게도 가능할거구요
```suggestion
  export interface DefaultTheme extends Theme {}
```

Reply:
오 나도 저거 조금 살짝 맘에 안들었었는데.. 감사합니다!

Reply:
흠.. 이거 적용해보려고 했는데 그래도 인텔리센스가 안 먹네요.. 좀 더 고민해봐야겠어요